htdocs/web_portal/javascript/service_group/add_ses_to_vsite.js

```diff
@@ -41,7 +41,7 @@ var resultsTable = {
 			addRow(colour, ses[i], table, plusCell(ses[i]));
 			// Swap the row colour
 			if (colour == "") {
-				colour = "1";
+				colour = "even";
```

Comment:
where is the colour corresponding to even defined?

Reply:
https://github.com/GOCDB/gocdb/blob/2feb5f498ad8fa37e6d29eb87ca7e1bccb23f2e2/htdocs/web_portal/css/web_portal.php#L344.

The colour is referring to the  even colour(As above). Where even is referring to the value of the Colour.